lesson1/src/stacks_queues/stack.js

```diff
@@ -37,7 +44,11 @@ class Stack {
    Should return a boolean indicating if queue is empty
    */
   isEmpty() {
-    // todo: do stuff
+    if (this.top == 0) {
```

Comment:
😝 this can be greatly simplified to just return ```return this.top === 0``` since that expressionevaluatess to a boolean